website/orders/services.py

```diff
@@ -86,7 +88,7 @@ def add_order(product, shift, order_type, user=None, set_done=False, force=False
             pass
         # Shift checks
         if not shift.can_order:
-            raise OrderException("User can not order products for this shift")
+            raise OrderException("This shift is closed.")
         if not shift.is_active:
             raise OrderException("This shift is not active")
```

Comment:
Isn't this checked by the model?

Reply:
We have to rewrite this thing anyway, as it does not work completely (see the TODO somewhere above this line) so let's do this in a different pull request.